tests/test_driver_alchemy_crud.py

```diff
@@ -845,69 +852,72 @@ def test_driver_get_all_versions_with_no_record():
 
             conn.execute(
                 """
-                INSERT INTO index_record(did, baseid, rev, form, size) VALUES (?,?,?,?,?)
-            """,
-                (did, baseid, rev, form, size),
+                INSERT INTO index_record(did, baseid, rev, form, size) VALUES ('{}','{}','{}','{}','{}')
+            """.format(
+                    did, baseid, rev, form, size
+                ),
             )
 
-        conn.commit()
-
         with pytest.raises(NoRecordFound):
             driver.get_all_versions("some baseid")
 
 
-@util.removes("index.sq3")
-def test_driver_get_fails_with_invalid_id():
+def test_driver_get_fails_with_invalid_id(combined_default_and_single_table_settings):
     """
     Tests retrieval of a record fails if the record id is not found.
     """
-    with sqlite3.connect("index.sq3") as conn:
-        driver = SQLAlchemyIndexDriver("sqlite:///index.sq3")
+    engine = create_engine(POSTGRES_CONNECTION)
+    driver = SQLAlchemyIndexDriver(POSTGRES_CONNECTION)
 
+    with engine.connect() as conn:
         did = str(uuid.uuid4())
         baseid = str(uuid.uuid4())
         rev = str(uuid.uuid4())[:8]
         form = "object"
+        size = 512
+
+        conn.execute("INSERT INTO base_version(baseid) VALUES ('{}')".format(baseid))
 
         conn.execute(
             """
-            INSERT INTO index_record(did, baseid, rev, form, size) VALUES (?,?,?,?,?)
-        """,
-            (did, baseid, rev, form, None),
+            INSERT INTO index_record(did, baseid, rev, form, size) VALUES ('{}','{}','{}','{}','{}')
+        """.format(
+                did, baseid, rev, form, size
+            ),
         )
 
-        conn.commit()
-
         with pytest.raises(NoRecordFound):
             driver.get("some_record_that_does_not_exist")
 
 
-def test_driver_update_record(skip_authz):
+def test_driver_update_record(skip_authz, combined_default_and_single_table_settings):
     _test_driver_update_record()
 
 
-@util.removes("index.sq3")
 def _test_driver_update_record():
     """
     Tests updating of a record.
     """
-    with sqlite3.connect("index.sq3") as conn:
```

Comment:
"Deployment changes" should mention that sqlite databases are not supported anymore